docs/admin/support-links.md

```diff
@@ -0,0 +1,28 @@
+# Support Links
+
+Support links let admins adjust the user dropdown menu to include links referring to internal company resources. The menu section replaces the original menu positions: documentation, report a bug to GitHub, or join the Discord server.
+
+![support links](../images/admin/support-links.png)
```

Comment:
In this context, we should probably show the default support links.

Reply:
Fixed!